xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -257,8 +265,7 @@
         <param name="utf8Destination">The destination buffer to which the encoded form of <paramref name="utf8Source" /> will be written.</param>
         <param name="bytesConsumed">The number of bytes consumed from the <paramref name="utf8Source" /> buffer.</param>
         <param name="bytesWritten">The number of bytes written to the <paramref name="utf8Destination" /> buffer.</param>
-        <param name="isFinalBlock">
-          <see langword="true" /> if there is further source data that needs to be encoded; otherwise, <see langword="false" />.</param>
+        <param name="isFinalBlock"><see langword="false" /> if there is further source data that needs to be encoded; otherwise, <see langword="true" />.</param>
```

Comment:
Since `isFinalBlock=true` is the default, I think we should state that first (rather than false). Combining with @Jozkee's feedback and some re-wording.

```suggestion
        <param name="isFinalBlock"><see langword="true" /> to indicate the source buffer contains the entire data to encode; otherwise, <see langword="false" />.</param>
```